index.html

```diff
@@ -18,69 +18,72 @@
       <!--------- Header section -------->
       <header>
         <!-- Navbar -->
+
         <nav>
-          <a href="#"
-            ><img
-              class="page-logo"
-              src="images/Logo.png"
-              alt="Website logo containing the words, 'My Logo'"
-          /></a>
-          <a href="#"
-            ><img
-              class="page-menu"
-              src="images/menu.png"
-              alt="The menu icon leading to the options: 'Portfolio', 'About' and 'Contacts'"
-          /></a>
+          <a class="nav__brand" href="#">Chris Mutugi</a>
+
           <ul class="nav__list">
-            <li class="nav__list-item"><a href="#">Portfolio</a></li>
-            <li class="nav__list-item"><a href="#">About</a></li>
-            <li class="nav__list-item"><a href="#">Contacts</a></li>
+            <li class="nav__item">
+              <a href="#works" class="nav__link">Portfolio</a>
+            </li>
+            <li class="nav__item">
+              <a href="#about-myself-wrapper" class="nav__link">About</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contacts-container" class="nav__link">Contacts</a>
+            </li>
```

Comment:
- [ ] You did well working on the mobile menu however you still have to add an (X) button to close the menu as required and as per the Figma design.

Your Project:
![review](https://user-images.githubusercontent.com/53347966/223483294-60181b8a-c3ff-44fa-84b3-ffb73723913f.JPG)